README.rst

```diff
@@ -137,6 +144,25 @@ You can instantiate quantities with one of four methods:
    torque.units.name  # "N m"
    torque.si_units  # "kg m^2 s^-2"
 
+With ``NumPy`` installed, you can instantiate a ``Quantity`` using either
+a list of floats or a ``NumPy`` array:
+
+.. code:: python
+
+    from ansys.units import Quantity
+    import numpy as np
+
+    length_array_quantity = Quantity(value=[1.0, 6.0, 7.0], units="m")
+
+    length_array = np.array([1.0, 6.0, 7.0])
+    length_array_quantity = Quantity(value=values, units="m")
```

Comment:
@seanpearsonuk `values` in line 158 is not defined.

![image](https://github.com/ansys/pyansys-units/assets/106588300/20520b5f-2c18-440d-912d-0229cd482669)